Dockerfile.assisted-test-infra

```diff
@@ -20,6 +20,7 @@ RUN dnf -y install \
   libguestfs-tools \
     && dnf config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo \
     && dnf -y install packer \
```

Comment:
hmm, but you already overridden the original packet binary with this command, no?
how can that help?

Reply:
It install it in a different location:
Hashcorp packer installed to: /usr/bin/packer
and the other to: /usr/sbin/packer
Anyway this is a known issue.:
Please see: https://learn.hashicorp.com/tutorials/packer/get-started-install-cli#troubleshooting

Reply:
After renaming the Hashcorp packer:
[root@985a4ebc5f1d assisted-test-infra]# which packer
/usr/sbin/packer
[root@985a4ebc5f1d assisted-test-infra]# which packer.io
/usr/bin/packer.io

Reply:
Ack, thought they are writing to the same locations